x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -26,7 +27,8 @@ export enum DEFAULT_MODEL_MEMORY_LIMIT {
   classification = '100mb',
 }
 
-export const DEFAULT_NUM_TOP_FEATURE_IMPORTANCE_VALUES = 2;
+export const DEFAULT_NUM_TOP_FEATURE_IMPORTANCE_VALUES = 0;
```

Comment:
Just wondering why the default was changed from `2` to `0`?

Reply:
This was suggested by @valeriy42 since for large datasets feature importance calculation can be very expensive. The suggestion was to set it to 0 by default.

Reply:
The default value for the parameter in the json config is `0` as well.